spec/features/super_admin/logging/view_logs_for_ip_spec.rb

```diff
@@ -0,0 +1,88 @@
+describe 'View authentication requests for an IP', type: :feature do
+  let(:ip) { '1.2.3.4' }
+  let(:username) { 'ABCDEF' }
+
+  let(:admin_user) { create(:user, organisation_id: organisation.id) }
+  let(:organisation) { create(:organisation) }
+  let(:location) { create(:location, organisation_id: organisation.id) }
+
+  let(:other_user) { create(:user, organisation_id: other_user_organisation.id) }
+  let(:other_user_organisation) { create(:organisation) }
+  let(:other_user_location) { create(:location, organisation_id: other_user_organisation.id) }
+
+  let(:super_admin) { create(:user, super_admin: true) }
+  let(:super_admin_organisation) { create(:organisation) }
+  let(:super_admin_location) { create(:location, super_admin_organisation: super_admin.organisation) }
+
+  before do
+    Session.create!(
+      start: 3.days.ago,
+      username: username,
+      siteIP: ip,
+      success: true
+    )
+
+    Session.create!(
+      start: 3.days.ago,
+      username: username,
+      siteIP: ip,
+      success: true
+    )
+
+    Session.create!(
+      start: 3.days.ago,
+      username: username,
+      siteIP: ip,
+      success: true
+    )
+
+    create(:ip, location_id: location.id, address: ip)
+
+    sign_in_user super_admin
+  end
+
+  context 'when searching for an IP address' do
+    before do
+      visit ip_new_logs_search_path
+      fill_in "IP address", with: search_string
+      click_on "Show logs"
+    end
+
+    context "with a correct IP" do
+      let(:search_string) { ip }
+
+      it 'displays all results for the searched IP' do
+        expect(page).to have_content("Found 3 results for \"#{ip}\"")
+      end
+    end
+
+    context 'when a user from an organisation searches logs for an IP from another organisation' do
```

Comment:
This doesn't make sense to me, this currently reads like:
```
in `/super_admin/logging/`
'View authentication requests for an IP'
  'when searching for an IP address'
    'when a user from an organisation searches logs for an IP from another organisation'
      'displays all results for the searched IP'
```
When reading the nested contexts all in a row it should make sense. This is actually the output you see when you run `rspec` in the command line.

I'm confused by:
- the fact this is a user without admin privileges, yet its nested in the same context as a signed in super admin user and its in the super_admin folder.
- The wording
- Also it suggests that a non-super admin user should be able to see logs from another organisation which is incorrect.

I would suggest that this is better in the non-super admin spec. As an idea something like:
```
in `specs/features/logging/view_auth_requests_for_an_ip_spec.rb`
"when searching for an IP address"
  "when entering an IP address that belongs to another organisation"
    "does not display any logs"
```